yang/frr-nexthop.yang

```diff
@@ -236,7 +236,7 @@ module frr-nexthop {
 
   grouping nexthop-grouping {
     list nexthop {
-      key "nh-type gateway interface";
+      key "nh-type vrf gateway interface";
```

Comment:
Please add `min-elements 1;` to prevent routes without nexthops from being configured (e.g. via gRPC).

Reply:
done